transcriptome-assembly.md

```diff
@@ -0,0 +1,122 @@
+# Transcriptome assembly - some basics
+
+Learning objectives:

```

Comment:
are these accurate? your comment implied they were missing

Reply:
Yes it is accurate. I simply copied the learning objectives from the genome assembly, so they should be re-assessed.